bodhi/server/services/updates.py

```diff
@@ -571,16 +571,16 @@ def new_update(request):
                 for update in updates:
                     # Validate that <koji_tag>-pending-signing and <koji-tag>-testing exists,
                     # if not create them
-                    side_tag_pending_signing = update.release.get_pending_signing_side_tag(
+                    side_tag_signing_pending = update.release.get_pending_signing_side_tag(
                         from_tag)
-                    side_tag_testing = update.release.get_testing_side_tag(from_tag)
-                    if not koji.getTag(side_tag_pending_signing):
-                        koji.createTag(side_tag_pending_signing, parent=from_tag)
-                    if not koji.getTag(side_tag_testing):
-                        koji.createTag(side_tag_testing, parent=from_tag)
-
-                    to_tag = side_tag_pending_signing
-                    # Move every new build to <from_tag>-pending-signing tag
+                    side_tag_testing_pending = update.release.get_testing_side_tag(from_tag)
+                    if not koji.getTag(side_tag_signing_pending):
+                        koji.createTag(side_tag_signing_pending, parent=from_tag)
+                    if not koji.getTag(side_tag_testing_pending):
+                        koji.createTag(side_tag_testing_pending, parent=from_tag, perm="autosign")
```

Comment:
the CreateTag api from koji expect a perm_id (integer) and not a permission name. The only API that seems to allow to pass a permission name is editTag2.

```suggestion
                        koji.createTag(side_tag_testing_pending, parent=from_tag)
                        koji.editTag2(side_tag_testing_pending, perm="autosign")
```

Reply:
https://github.com/fedora-infra/bodhi/pull/3631/files#diff-14b55177c4dbffd842bef13afbea8a2aL506-R507
looked like it allowed for names

Reply:
> https://github.com/fedora-infra/bodhi/pull/3631/files#diff-14b55177c4dbffd842bef13afbea8a2aL506-R507
> looked like it allowed for names

Ok this is the dummy buildsystem that you linked here, so indeed we can accept anything here :-)